src/pages/Messages.tsx

```diff
@@ -1,39 +1,53 @@
 import { Box, Button, Divider, Stack, Typography } from "@mui/material";
 import ConversationList from "../components/Messages/ConversationList";
+import NavBar from "../components/NavBar/NavBar";
 
 const styles = {
   button: {
     marginTop: 1,
   },
-  root: {
-    width: "100%",
+  container: { height: "auto", justifyContent: "center" },
+  divider: { height: "auto" },
+  middleContent: { flex: "0 0 350px", height: "100vh", minWidth: 0 },
+  nav: { flex: "0 0 275px", height: "100vh", position: "sticky", top: 0 },
+  rightContent: {
+    height: "100vh",
+    flex: "0 0 600px",
   },
   selectMessageContainer: {
     display: "flex",
-    flexDirection: "column",
+    height: "100%",
     justifyContent: "center",
     alignItems: "center",
-    width: "50%",
   },
 };
 
 const Messages = () => {
   return (
-    <Stack direction="row" sx={styles.root}>
-      <ConversationList />
-      <Divider flexItem orientation="vertical" />
-      <Box sx={styles.selectMessageContainer}>
-        <Box>
-          <Typography variant="h6">Select a Message</Typography>
-          <Typography>
-            Choose one of your existing conversations or start a new one!
-          </Typography>
-          <Button variant="contained" sx={styles.button}>
-            New Message
-          </Button>
+    <Stack
+      direction="row"
+      divider={<Divider orientation="vertical" sx={styles.divider} />}
+      sx={styles.container}
+    >
+      <Box component="header" sx={styles.nav}>
+        <NavBar />
+      </Box>
+      <Box sx={styles.middleContent}>
+        <ConversationList />
+      </Box>
+      <Box sx={styles.rightContent}>
+        <Box sx={styles.selectMessageContainer}>
+          <Box>
+            <Typography variant="h6">Select a Message</Typography>
+            <Typography>
+              Choose one of your existing conversations or start a new one!
+            </Typography>
+            <Button variant="contained" sx={styles.button}>
+              New Message
+            </Button>
+          </Box>
         </Box>
       </Box>
```

Comment:
Similar to my comment on the `DirectMessages` page, I think adding a `Divider` on the RHS (of the Chat container) would be beneficial. It makes the app look more centered, rather than one component of the page being extremely larger than the rest

Reply:
As discussed offline, there's some complication around this since it adds an extra pixel, which makes the layout of the messages pages 1 pixel off compared to the other pages. Hopefully we can solve this in another ticket

Reply:
Same here as in `DirectMessage.tsx`